### PR TITLE
Autoscale: Add to official release and do renames

### PR DESCRIPTION
**Pull request by j82w**
# Pull Request Template

## Description

This changing the property names based on feedback.

Create throughput is the same:
`ThroughputProperties throughputProperties = ThroughputProperties.CreateAutoscaleThroughput(6000);`

New property name:
`throughputProperties.AutoscaleMaxThroughput`

Old property name:
`throughputProperties.MaxAutoscaleThroughput`

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### CODE REVIEW

File: Microsoft.Azure.Cosmos/src/Resource/Settings/ThroughputProperties.cs [hunk 1]
```diff
@@ -115,7 +115,7 @@ static ThroughputProperties CreateManualThroughput(int throughput)
 #else
         internal
 #endif
-        static ThroughputProperties CreateAutoscaleThroughput(
+        static ThroughputProperties CreateAutoscaleMaxThroughput(
             int maxAutoscaleThroughput)
```
Comment (bchong95) [inline Microsoft.Azure.Cosmos/src/Resource/Settings/ThroughputProperties.cs:119]: we need to argument check this that the value is not zero or negative.

File: Microsoft.Azure.Cosmos/tests/Microsoft.Azure.Cosmos.EmulatorTests/CosmosThroughputTests.cs [hunk 2]
```diff
@@ -19,6 +19,7 @@ public class CosmosThroughputTests
         public void TestInit()
         {
             this.cosmosClient = TestCommon.CreateCosmosClient();
+            //this.cosmosClient = new CosmosClient()
```
Comment (bchong95) [inline Microsoft.Azure.Cosmos/tests/Microsoft.Azure.Cosmos.EmulatorTests/CosmosThroughputTests.cs:22]: don't leave commented code.

Comment (j82w) [reply Microsoft.Azure.Cosmos/tests/Microsoft.Azure.Cosmos.EmulatorTests/CosmosThroughputTests.cs:22]: That's my bad. To run all the test I have to point it to prod currently.

**Comment by deborahc** [comment]
Looks good to me.

**Pull request merged**